homeassistant/components/devolo_home_control/config_flow.py

```diff
@@ -28,22 +28,22 @@ class DevoloHomeControlFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
     def __init__(self):
         """Initialize devolo Home Control flow."""
         self.data_schema = {
-            vol.Required(CONF_USERNAME): str,
+            vol.Required(CONF_EMAIL): str,
```

Comment:
Why are we changing the key being used for the config flow? This is a breaking change and I don't see what value it adds

Reply:
I thought, that was what SNoof85 requested? I don't have strong feelings about the name, but about not breaking anything, so I reverted it.

Reply:
Mhh sorry, no that was not the intention.
My comment was just on the key used. Choosing either username or email. I did not saw i was also changed to email to the whole integration ! My bad.